src/transformers/models/tapas/modeling_tapas.py

```diff
@@ -43,7 +43,6 @@
 logger = logging.get_logger(__name__)
 
 _CONFIG_FOR_DOC = "TapasConfig"
-_TOKENIZER_FOR_DOC = "TapasTokenizer"
 _TOKENIZER_FOR_DOC = "google/tapas-base"
```

Comment:
`_TOKENIZER_FOR_DOC = "google/tapas-base"`  one line below should be deleted too.